aio/src/app/shared/search-results/search-results.component.html

```diff
@@ -1,30 +1,59 @@
 <div class="search-results">
-  <div *ngIf="searchAreas.length; then searchResults; else notFound"></div>
-</div>
+  <div>
+    <div [ngSwitch]="searchState">
+
+      <div *ngSwitchCase="1">
```

Comment:
Let's avoid unnecessary DOM elements (here and below):

```suggestion
      <ng-container *ngSwitchCase="1">
```